src/center.jl

```diff
@@ -85,11 +85,10 @@ function center!(X::AbstractMatrix,
     center!(X, ObsDim.Constant{2}(), operate_on)
 end
 
-function center!(X::AbstractMatrix,
+function center!(X::AbstractArray,
```

Comment:
what happens for a 3-d (or more) array? `\mu` is always a vector

Reply:
I'm not sure, I think this change was inadvertant

Reply:
reverted